### PR TITLE
23 example outputs

### LINKED ISSUE #23: Example outputs
Make a python file with several example files and dictionaries, and record outputs.

### PR DESCRIPTION
**Pull request by VaradK62442**
simple ipynb showing HR, SM, SPA usage.

lmk if anything else should be added

### CODE REVIEW

File: examples.ipynb [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+    "file_locations = \"example_files\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Hospital Residents Problem\n",
+    "\n",
+    "Both the file and dictionary are ran through both the hospital oriented and resident oriented version of the algorithm."
```
Comment (BenOnTheBoard) [inline examples.ipynb]: Run through

File: examples.ipynb [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+    "assert sm_1_woman_stable_matching == sm_2_woman_stable_matching"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Student Project Allocation\n",
+    "\n",
+    "We follow a similar process. Both the `spa1.txt` file and the dictionary defined are ran through both the student and lecturer oriented version of the algorithm."
```
Comment (BenOnTheBoard) [inline examples.ipynb]: run through

File: examples.ipynb [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+    "Since both the dictionary and the file have the same HR instance, you can see that they both evaluate to the same stable matching."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stable Marriage\n",
+    "\n",
+    "We follow a similar process. Both the `sm1.txt` file and the dictionary defined are ran through both the man and woman oriented version of the algorithm."
```
Comment (BenOnTheBoard) [inline examples.ipynb]: run through

**Review by BenOnTheBoard — CHANGES_REQUESTED** [review]
This all runs for me. I like the shortened imports.
Just a few typos.

**Review by BenOnTheBoard — COMMENTED** [review]
Great. Everything's ready.

**Comment by VaradK62442** [comment]
fantastic

**Pull request merged**